Client/Config/WriteOptions.cs

```diff
@@ -0,0 +1,22 @@
+using InfluxDB3.Client.Write;
+
+namespace InfluxDB3.Client.Config;
+
+public class WriteOptions
```

Comment:
Please add doc string for `WriteOptions`:

```suggestion
/// <summary>
/// The WriteOptions class holds the configuration for writing data to InfluxDB.
///
/// You can configure following options:
/// - Precision: The default precision to use for the timestamp of points if no precision is specified in the write API call.
/// - GzipThreshold: The threshold in bytes for gzipping the body. The default value is 1000.
///
/// If you want create client with custom options, you can use the following code:
/// <code>
/// using var client = new InfluxDBClient(new InfluxDBClientConfigs{
///     HostUrl = MockServerUrl,
///     Organization = "my-org",
///     Database = "my-database",
///     WriteOptions = new WriteOptions
///    {
///        Precision = WritePrecision.S,
///         GzipThreshold = 4096
///    }
/// }); 
/// </code>
/// </summary>
public class WriteOptions
```